gdk/wizard/WizardChecker.py

```diff
@@ -77,3 +83,70 @@ def is_valid_custom_build_command(self, input_value):
             return False
         except ValueError:
             return len(input_value) > 0
+
+    def check_build_options(self, input_value):
+        try:
+            # Convert the input string to a dictionary (object)
+            input_obj = json.loads(input_value)
+        except json.JSONDecodeError:
+            return False
+
+        if not isinstance(input_obj, dict):
+            return False
+
+        if len(input_obj) > 2:
+            return False
+
+        if "excludes" not in input_obj or "zip_name" not in input_obj:
+            return False
+
+        excludes = input_obj.get("excludes")
+        if (
+            not isinstance(excludes, list)
+            or len(excludes) < 1
+            or not all(isinstance(item, str) for item in excludes)
+        ):
+            return False
+
+        zip_name = input_obj.get("zip_name")
+        if not isinstance(zip_name, str):
+            return False
+
+        return True
+
+    def check_bucket(self, input_value):
+        # input must be a non-empty string
+        return isinstance(input_value, str) and len(input_value) > 0
+
+    def check_region(self, input_value):
+        # input must be a non-empty string
+        return isinstance(input_value, str) and len(input_value) > 0
+
+    def check_publish_options(self, input_value):
+        # input_value will always be a string so must try to convert it to a dict
+
+        try:
+            input_object = json.loads(input_value)
+        except json.JSONDecodeError:
+            return False
+
+        if not isinstance(input_object, dict):
+            return False
+
+        if "file_upload_args" in input_object and isinstance(
+            input_object["file_upload_args"], dict
+        ):
+            return True
+
+        return False
+
+    def check_gdk_version(self, input_value):
+        gdk_version_pattern = (
+            "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)"
+            "(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\."
+            "(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?"
+            "(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?"
+        )
+        if re.match(gdk_version_pattern, input_value):
+            return True
+        return False
```

Comment:
```suggestion
        return re.match(gdk_version_pattern, input_value)
```

Reply:
I tried this at first it doesn't work unfortunately